python/ribasim/ribasim/model.py

```diff
@@ -182,14 +182,21 @@ def _save(self, directory: DirectoryPath, input_dir: DirectoryPath):
             sub._save(directory, input_dir)
 
     def set_crs(self, crs: str) -> None:
-        self.crs = crs
-        self.edge.df = self.edge.df.set_crs(crs)
+        self._apply_spatial_function("set_crs", crs)
+
+    def to_crs(self, crs: str) -> None:
+        self.set_crs(self.crs)
```

Comment:
This should probably be removed, or at least be called *after* to_crs, if that doesn't set the crs of all geodataframes.

Now this would change any previous crs to this crs, and then ask to project to that crs, resulting in a nothing/identity transform?

Reply:
As discussed, there was some confusion about what this line does. 
I've added a comment and will open a tech-dept issue.